src/components/security_manager/test/crypto_manager_impl_test.cc

```diff
@@ -131,18 +137,23 @@ TEST_F(CryptoManagerTest, WrongInit) {
   // We have to cast (-1) to security_manager::Protocol Enum to be accepted by
   // crypto_manager_->Init(...)
   // Unknown protocol version
-  security_manager::Protocol UNKNOWN =
-      static_cast<security_manager::Protocol>(-1);
-
   EXPECT_CALL(*mock_security_manager_settings_, security_manager_mode())
-      .WillRepeatedly(Return(security_manager::SERVER));
-  EXPECT_CALL(*mock_security_manager_settings_,
-              security_manager_protocol_name()).WillOnce(Return(UNKNOWN));
+      .WillOnce(Return(security_manager::SERVER));
+  EXPECT_CALL(*mock_security_manager_settings_, force_unprotected_service())
+      .WillOnce(ReturnRef(is_services));
+  EXPECT_CALL(*mock_security_manager_settings_, force_protected_service())
+      .WillOnce(ReturnRef(is_services));
   EXPECT_FALSE(crypto_manager_->Init());
-
+  // const std::vector<int> ser;
```

Comment:
@ValeriiMalkov Commented code?

Reply:
@AByzhynar Fixed at the commit e54a9eed3db7d43eb84669db62e5cb5bc7ec8a87